UPGRADE-6.3.md

```diff
@@ -72,11 +107,18 @@ is recommended (although this tool may not find database-related deprecations).
 
 ## Services
 
-None.
+- chameleon_system_core.pdo
```

Comment:
Also chameleon_system_core.sessionPdo?

The service "database_connection" is available from Doctrine now?

Reply:
Nope (is still required, but is now initialized differently) and yes.